AnkiDroid/src/main/java/com/ichi2/libanki/sched/AbstractSched.java

```diff
@@ -178,29 +178,25 @@
      * this field and use getNamePart(0) for those cases.
      */
     public class DeckDueTreeNode implements Comparable {
-        private String[] mName;
+        private final String mName;
+        private String[] mSplittedName;
```

Comment:
I like this change a lot I think it makes it more clear to reason about the name, certainly.
Now I'm going to propose a change that will touch every line of the PR just about, but is also a simple rename refactor in Android Studio :-), "Splitted" is not a valid word in English, and the variable really represents an array of the variable's compoments, so perhaps:

```suggestion
        private String[] mNameComponents;
```

Reply:
Done